ops/terraform/env/mgmt/s3.tf

```diff
@@ -119,3 +119,57 @@ resource "aws_s3_bucket_policy" "logging" {
 }
 POLICY
 }
+
+resource "aws_s3_bucket" "bfd_public_test_data" {
+  bucket = "bfd-public-test-data"
+}
+
+resource "aws_s3_bucket_server_side_encryption_configuration" "bfd_public_test_data" {
+  bucket = aws_s3_bucket.bfd_public_test_data.id
+  rule {
+    apply_server_side_encryption_by_default {
+      sse_algorithm = "AES256"
+    }
+  }
+}
+
+resource "aws_s3_bucket_versioning" "bfd_public_test_data" {
+  bucket = aws_s3_bucket.bfd_public_test_data.id
+  versioning_configuration {
+    status = "Enabled"
+  }
+}
+
+resource "aws_s3_bucket_public_access_block" "bfd_public_test_data" {
+  bucket = aws_s3_bucket.bfd_public_test_data.id
+
+  block_public_acls       = true
+  block_public_policy     = true
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+}
+
+resource "aws_s3_bucket_ownership_controls" "bfd_public_test_data" {
+  bucket = aws_s3_bucket.bfd_public_test_data.id
+  rule {
+    object_ownership = "BucketOwnerEnforced"
+  }
+}
+
+resource "aws_s3_bucket_lifecycle_configuration" "bfd_public_test_data" {
+  depends_on = [aws_s3_bucket_versioning.bfd_public_test_data]
```

Comment:
Is this explicit `depend_on` necessary? I think the terraform DAG will take care of this, given the reference to the bucket by address just below, right?

Reply:
it is not, I removed it.